packages/genui/a2ui-playground/src/pages/AIChatPage.css

```
@@ -760,45 +763,52 @@
-  background: color-mix(in srgb, var(--geist-foreground) 4%, transparent);
+.chatMessageChunkJson {
+  margin: 0;
+  max-height: 260px;
+  overflow: auto;
+  padding: 8px 10px;
+  color: var(--geist-secondary);
+  font-family: var(--geist-mono);
+  font-size: 12px;
+  line-height: 1.55;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)